docs/cli.md

```diff
@@ -0,0 +1,71 @@
+# CLI Documentation for iXdiagnose
+
+`iXdiagnose` is a command-line tool designed to generate debug information on TrueNAS systems. This documentation provides an overview of the available commands, options, and their usage.
+
+## Installation
+
+Before using `iXdiagnose`, make sure it's properly installed. You can follow the installation steps in the [build docs](https://github.com/truenas/ixdiagnose/blob/master/docs/build.md) if available.
```

Comment:
No reason for the `Installation` section since we ship this by default starting in Cobia.